gcloud-java-storage/src/main/java/com/google/gcloud/storage/Storage.java

```diff
@@ -566,6 +622,22 @@ public static BlobGetOption metagenerationNotMatch(long metageneration) {
     public static BlobGetOption fields(BlobField... fields) {
       return new BlobGetOption(StorageRpc.Option.FIELDS, BlobField.selector(fields));
     }
+
+    static BlobGetOption[] setGeneration(BlobId blobId, BlobGetOption... options) {
```

Comment:
Same comment as above. I don't understand why we would need this `setGeneration` method.